content/episodes/2018-08-13-S03E04-virtual-connections.md

```diff
@@ -0,0 +1,243 @@
+---
+categories: "episodes"
+date: 2018-08-08T04:55:00-08:00
+description: ":'-) Ever wonder why emoticons exist? They popped up in the 1980s to make online connections feel a little less digital and a little more personal :D. In this episode of IRL, host [Veronica Belmont](https://twitter.com/Veronica/) and special guest [Peter Rojas](https://twitter.com/peterrojas/)  explore how the Internet is both building and also confusing our relationships every day. [Chloe Stuart-Ulin](https://twitter.com/chloerosewrites/) gives a first-hand account of her life as a “closer” for an online-dating service; we hear a dramatic, real-life story about a woman who finds her biological parent online; and [Emma Brockes](https://www.theguardian.com/technology/series/digital-dilemmas/) talks about how we can all maintain humanity while interacting with others on the internet."
+embed: "588ea281"
+mp3: "https://dts.podtrac.com/redirect.mp3/audio.simplecast.com/588ea281.mp3"
+number: 19
+ogdescription: "Veronica Belmont and Peter Rojas explore how the Internet is building and confusing our relationships."
+ogtitle: "Virtual Connections"
+season: 3
+seasonepisode: 4
+shownotes: |
+    Read more about [Chloe Rose's experience as a "closer" for hire](https://qz.com/1247382/online-dating-is-so-awful-that-people-are-paying-virtual-dating-assistants-to-impersonate-them/) on online dating apps.
+
+    Emma Brockes writes a column for the Guardian called [How to be Human Online](https://www.theguardian.com/technology/series/digital-dilemmas/). She's just written a book too called, [An Excellent Choice: Panic and Joy on My Solo Path to Motherhood](https://emmabrockes.com/about/).
+
+    Read [Ingrid Burrington's essay about CorrLinks](http://blog.mozilla.org/internetcitizen/2018/08/08/the-banality-of-prison-email/),
+    the email service providing connection for inmates at U.S. prisons.
+
+    Check out this article about [how the internet has changed dating forever](http://blog.mozilla.org/internetcitizen/2018/08/08/how-the-web-changed-dating-forever/).
+    Online dating coach Laurie Davis Edward shares her thoughts on the good, bad and ugly that comes with finding love on the web.
+
+    And, for more about human connection, and what our innate desire for it
+    means for us as we — more and more — love, do business, and find our tribes
+    online, [read this piece by cultural anthropologist Genevieve Bell](http://blog.mozilla.org/internetcitizen/2018/08/13/human-innate-connection-genevieve-bell/).
+
+    Finally, for some bonus audio on [how technology interfered with a marriage
+    proposal](http://blog.mozilla.org/internetcitizen/2018/08/13/online-relationships/) — and commentary on new relationship norms — head over to [Mozilla's blog](http://blog.mozilla.org/internetcitizen/2018/08/13/online-relationships/).
+url: /season4/episode04/
```

Comment:
🙊 

Perhaps a problem with `archetypes/episodes.md`?

Reply:
I screwed up typing the file name. I thought I'd found all the places where it was wrong. Sorry to bother you for something so straight forward 😖

Reply:
No worries! Bugs like this are the cost of automation. You _assume_ it's fine.